src/components/NumberInput/NumberInput.css

```diff
@@ -0,0 +1,28 @@
+.NumberInput {
+	margin-bottom: 32px;
+}
+
+.NumberInput label {
+	font-size: 14px;
+	line-height: 16px;
+	font-weight: bold;
+	margin-bottom: 6px;
+	display: block;
+}
+
+.NumberInput input {
+	background: #1B1B26;
+	padding: 12px 16px;
+	color: #fff;
+	font-size: 16px;
+	line-height: 24px;
+	border: 0;
```

Comment:
This line is no longer relevant as you're setting the border width to ``1px`` a few lines down

Reply:
Hmm - I just ripped this from the TextInput. - I'll put the changes back in there as well.